shoper.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pandas as pd
+from operator import itemgetter
+from re import search
+
+
+def deactivate_products(database, dc_price, replacement_price):
+    """
+    Deactivates products set with exact price - e.g. 107.99
+    Products are kept because some of them have been withdrawn,
```

Comment:
To jest jasne z kontekstu nazwy funkcji (co może się zmienić zależnie od tego, jak zostaną zaadresowane poprzednie komentarze).